### PR TITLE
feat(editing-support): add auto-save.nvim plugin

### PR DESCRIPTION
**Pull request by rozhelluk**

### CODE REVIEW

File: lua/astrocommunity/editing-support/auto-save-nvim/auto-save-nvim.lua [hunk 1]
```diff
@@ -0,0 +1,5 @@
+return {
+  "Pocco81/auto-save.nvim",
+  lazy = false,
```
Comment (luxus) [inline lua/astrocommunity/editing-support/auto-save-nvim/auto-save-nvim.lua:3]: why did you turn off lazy?

File: lua/astrocommunity/editing-support/auto-save-nvim/auto-save-nvim.lua [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  config = function() require("auto-save").setup {} end,
```
Comment (luxus) [inline lua/astrocommunity/editing-support/auto-save-nvim/auto-save-nvim.lua:4]: any reason why you don't use `opts = {}`

**Review by luxus — CHANGES_REQUESTED** [review]

File: lua/astrocommunity/editing-support/auto-save-nvim/auto-save-nvim.lua (hunk 2 above)
Comment (mehalter) [reply lua/astrocommunity/editing-support/auto-save-nvim/auto-save-nvim.lua:4]: Moved to suggestion type comment.

**Review by mehalter — CHANGES_REQUESTED** [review]
Made a few suggestions

**Review by mehalter — APPROVED** [review]
Just to expedite this getting merged, I went ahead and applied the changes. Thanks again for these contributions! <3

**Pull request merged**